File: Kotlin/24.kt
fun main() {

    var num = ArrayList<Double>(5)
    num.add(3.0)
    num.add(89.03)
    num.add(6.4)
    num.add(2.21)
    num.add(4.4)
    var m = 0.0
    for (i in num){
        m += i
    }
    println(m/5)
    
}

File: Swift/24.swift
func hello(yourName a:String) {
    print("Hello",a)
}

hello(yourName: "Swift")

func plus(_ one: Int,_ two:Int) {
    print(one + two)
}

plus(1,4)

func ave(_ num: Int...) -> Double {
    var total = 0.0

    for i in num {
        total += Double(i)
    }

    return total / Double(num.count)
}

print(ave(1,123,12,32))

func minus(a: Int , b: Int = 4) {
    print(a+b)
}

minus(a: 3)
minus(a: 3,b:4)